### PR TITLE
Tweak build

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: code/build.R [hunk 1]
```diff
@@ -63,69 +62,69 @@ if (length(failed_regions) > 0) {
...
-    folder_path = paste0("outputdata/", parameters$date_routing,"/",  snakecase::to_snake_case(parameters$region), "/", "coherent_networks/")
+    folder_path = file.path(output_folder, region_snake, "coherent_networks")
 
     if(!dir.exists(folder_path)) {
       dir.create(folder_path, recursive = TRUE)
     }
-
-    for (city in parameters$coherent_area) { 
+    
+    open_roads_path = paste0("inputdata/OS_Scotland_Network_", region_snake, ".geojson")
```
Comment (Robinlovelace) [inline code/build.R:99]: Heads-up @wangzhao0217 this is the cause. Where is the input data?

It only has 2 columns:

```
>     open_roads_path = paste0("inputdata/OS_Scotland_Network_", region_snake, ".geojson")
>     open_roads_unprojected = sf::read_sf(open_roads_path)
>     open_roads = sf::st_transform(open_roads_unprojected, "EPSG:27700")
>     names(open_roads)
[1] "identifier" "geometry"

File: code/build.R [hunk 2]
```diff
@@ -63,69 +62,69 @@ if (length(failed_regions) > 0) {
...
-
-        min_percentile_value = stats::quantile(NPT_zone$all_fastest_bicycle_go_dutch, probs = parameters$coherent_percentile, na.rm = TRUE)
+        combined_net_zone = combined_net[city_boundary, ]
+        min_percentile_value = stats::quantile(
+          combined_net_zone$all_fastest_bicycle_go_dutch,
+          probs = parameters$coherent_percentile,
+          na.rm = TRUE
+        )
 
-        open_roads_national_zone = open_roads_national[sf::st_union(zone), , op = sf::st_intersects]
```
Comment (Robinlovelace) [inline code/build.R:119]: open_roads_national was not created on previous branch.

**Review by Robinlovelace — COMMENTED** [review]
Figured out why it's not reproducible at this stage.

**Review by wangzhao0217 — APPROVED** [review]
looks good

**Pull request merged**